dev/io.openliberty.org.apache.myfaces.4.0/src/org/apache/myfaces/renderkit/html/base/HtmlRadioRendererBase.java

```diff
@@ -164,8 +164,14 @@ else if (layout.equals(LAYOUT_LIST))
         {
             // Render as single component
             writer.startElement(usingTable != null ? HTML.TABLE_ELEM : HTML.UL_ELEM, selectOne);
-            HtmlRendererUtils.renderHTMLAttributes(writer, selectOne,
-                                                   HTML.SELECT_TABLE_PASSTHROUGH_ATTRIBUTES);
+            //TODO Committing back to myfaces under PR: https://github.com/apache/myfaces/pull/419
```

Comment:
It looks like we already merged this. Could you clean up this comment?

Reply:
Same as the previous comment.